### ISSUE TITLE
BouncerLink - Link Shortening but with Authentication & Notifications

### ISSUE THREAD

**Issue by omranjamal** [DevArifHossain/saas-idea-validator#4]
# BouncerLink

## Description
Imagine you have to send a link to someone over a public medium and you only want to allow that specific person to be forwarded to the actual link and you want to be notified instantly the moment they open the link.

That's where bouncer.link comes in. You shorten links and get access to these specific features. As well as a history of the number of times the link was opened.

### A possible side feature
The ability to send different people to different links upon logging in.

## Use Cases
- Timing people from the time they open the link.
- Sharing private links over public medium.
- Tracking and making sure everyone in an email thread has seen and reviewed the link.
- Tracking how many unique people open a link.
- Sending different people to different links (for sales, tracking & analytics).

**Comment by DevArifHossain** [comment]
I like this one! It's kind of like bit.ly but private/secure?

**Comment by omranjamal** [comment]
Yeah; pretty much. (Let's make it; Just one free project on Firebase and a nodejs hosting server required)

**Comment by omranjamal** [comment]
I think we can get it up in less than a week! We will invent nothing just mix and match tools and get things going.

**Comment by jaccinelli** [comment]
I think i don't fully get it.
1) Why would you need to share a private link in a public place? 
2) *Consider 1 just completely answered and validated* - If there is authentication and you authored that person (which i can asume that you know him then) what is the difference between private direct message over any platform?

Just asking because i didn't get it.
